config/local-development/tilt/replace-image.yaml

```diff
@@ -1,4 +1,8 @@
 - op: replace
   path: /spec/template/spec/containers/1/image
   value:
-    quay.io/raffaelespazzoli/patch-operator:latest
\ No newline at end of file
+    quay.io/raffaelespazzoli/patch-operator:latest
```

Comment:
This should point to `redhat-cop` org

Reply:
no this is for local development, it should stay that way